pkg/apis/pipeline/v1beta1/pipelinerun_validation_test.go

```diff
@@ -92,7 +92,7 @@ func TestPipelineRun_Invalidate(t *testing.T) {
 					Status: "PipelineRunCancell",
 				},
 			},
-			want: apis.ErrInvalidValue("PipelineRunCancell should be PipelineRunCancelled", "spec.status"),
+			want: apis.ErrInvalidValue("PipelineRunCancell should be PipelineRunCancelled or PipelineRunPending", "spec.status"),
```

Comment:
typo: PipelineRunCancel (one l)

Reply:
Yes, this is a previously existing test for an incorrect status, thus the typo.